### ISSUE TITLE
Coolmomentum optimizer

### ISSUE THREAD

**Issue by borbysh** [ifeherva/optimizer-benchmark#1]
Dear  Istvan!
Thank you for this project. Together with my colleague, we have recently developed a new stochastic gradient optimizer called Coolmomentum. It has demonstrated prospective results. Could you please benchmark it in your project?) Please see  https://github.com/borbysh/coolmomentum 
for details.

Regards
Oleksandr

**Comment by ifeherva** [comment]
Completed. Sorry for the long delay, was away from github. If there are interesting papers or new research you wanna see please let me know.

**Comment by ifeherva** [comment]
Also, what hyperparms did you use for your run? I get much worse results (see wandb dashboard).

**Comment by borbysh** [comment]
Hi! Could you please make a new run, taking 

https://github.com/borbysh/coolmomentum/blob/master/train_cifar10.py
https://github.com/borbysh/coolmomentum/tree/master/models
https://github.com/borbysh/coolmomentum/tree/master/optimizers

These files are taken from your original project and modified to use Coolmomentum optimizer. They already contain all hyperparams used to obtain results shown in our project

**Comment by borbysh** [comment]
Hi! Could you please make a new run, taking

https://github.com/borbysh/coolmomentum/blob/master/train_cifar10.py
https://github.com/borbysh/coolmomentum/tree/master/models
https://github.com/borbysh/coolmomentum/tree/master/optimizers

These files are taken from your original project and modified to use
Coolmomentum optimizer. They already contain all hyperparams used to obtain
results shown in our project


Regards

Oleksandr

**Comment by ifeherva** [comment]
Cool, I disabled milestones and used the provided hyper-params. Wandb board is updated.

**Comment by ifeherva** [comment]
Does Coolmomentum require no LR scheduling?

**Comment by borbysh** [comment]
Thank you! Yes, Coolmomentum uses the momentum coefficient scheduling
instead of the LR one. Btw, we didn't apply LR dropout in coolmomentum, did
you?

**Comment by ifeherva** [comment]
I did not, I followed the same configuration that you shared. The model performed very well.

**Comment by borbysh** [comment]
Ok, I'll try it in a while

**Comment by TomLucidor** [comment]
Mix in Lion and the other SOTA for the last 2 years pls
